rdbms/src/main/java/io/spine/server/storage/jdbc/type/JdbcColumnTypes.java

```diff
@@ -0,0 +1,251 @@
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package io.spine.server.storage.jdbc.type;
+
+import com.google.protobuf.AbstractMessage;
+import com.google.protobuf.Timestamp;
+import com.google.protobuf.util.Timestamps;
+import io.spine.core.Version;
+import io.spine.json.Json;
+import io.spine.server.storage.jdbc.DatabaseException;
+import io.spine.server.storage.jdbc.Sql;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * A factory for basic {@link JdbcColumnType} implementations.
+ *
+ * @author Alexander Aleksandrov
+ */
+final class JdbcColumnTypes {
+
+    private JdbcColumnTypes() {
+        // Prevent instantiation of a utility class
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@code boolean} columns
+     */
+    static JdbcColumnType<Boolean, ?> booleanType() {
+        return new BooleanColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@code String} columns
+     */
+    static JdbcColumnType<String, ?> stringType() {
+        return new StringColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@code int} columns
+     */
+    static JdbcColumnType<Integer, ?> integerType() {
+        return new IntegerColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@code long} columns
+     */
+    static JdbcColumnType<Long, ?> longType() {
+        return new LongColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@link Version} columns
+     */
+    static JdbcColumnType<Version, ?> versionType() {
+        return new VersionColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for {@link Timestamp} columns
+     */
+    static JdbcColumnType<Timestamp, ?> timestampType() {
+        return new TimestampColumnType();
+    }
+
+    /**
+     * @return new instance of {@link JdbcColumnType} for
+     * {@link AbstractMessage Message} columns
+     * @see io.spine.server.entity.storage.ColumnTypeRegistry#get
+     */
+    static JdbcColumnType<AbstractMessage, ?> messageType() {
+        return new MessageColumnType();
+    }
+
+    private static class BooleanColumnType extends SimpleJdbcColumnType<Boolean>{
+
+        @Override
+        public void setColumnValue(PreparedStatement storageRecord, Boolean value,
+                                   Integer columnIdentifier) {
+            try {
+                storageRecord.setBoolean(columnIdentifier, value);
+            } catch (SQLException e) {
+                throw new DatabaseException(e);
+            }
+
+        }
+
+        @Override
+        public Sql.Type getSqlType() {
+            return Sql.Type.BOOLEAN;
+        }
+
```

Comment:
Please kill some of redundant empty lines. There are lots in this file.

Also, similar to my request above, the nested classes like this one must be documented. Just re-iterating my request one more time.